### ISSUE TITLE
The engine "node" is incompatible with this module. 

### ISSUE THREAD

**Issue by mxschmitt** [qawolf/playwright-video#2]
Yarn install output:
```
➜  playwright-record-example yarn add -D playwright-video
yarn add v1.21.1
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
error playwright-video@0.1.4: The engine "node" is incompatible with this module. Expected version ">= 12.13 <13". Got "13.0.1"
error Found incompatible module.
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
```

I would only set it to `>= 12.13`, or what do you think? So that 13.x is also allowed.

**Comment by jperl** [comment]
Will fix this

**Comment by flaurida** [comment]
Thanks for reporting @mxschmitt!

**Comment by jperl** [comment]
Should be fixed in `0.1.5`
